modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowAssetPublisher.testcase

```diff
@@ -232,4 +232,76 @@ definition {
 			webContentTitle = "Blogs Entry Title");
 	}
 
+	@description = "This is a test for LRQA-69814. Add a basic web content with workflow via asset publisher and approve it. "
+    @ignore = "Test Stub"
+    @priority = "3"
+    test AssetPublisherApproveWC {
+
+         // TODO LRQA-69814 AssetPublisherApproveWC pending implementation
+
+    }
+
+    @description = "This is a test for LRQA-69815. Add a basic web content with workflow via asset publisher and reject it. "
+    @ignore = "Test Stub"
+    @priority = "3"
+    test AssetPublisherRejectWC {
+
+        // TODO LRQA-69815 AssetPublisherRejectWC pending implementation
+
+    }
+
+    @description = "This is a test for LRQA-69816. Add a blog with workflow via asset publisher and approve it. "
+    @ignore = "Test Stub"
+    @priority = "3"
+    test AssetPublisherApproveBlog {
+
+        // TODO LRQA-69816 AssetPublisherApproveBlog pending implementation
+
+    }
+
+    @description = "This is a test for LRQA-69817. Add a blog with workflow via asset publisher and reject it. "
+    @ignore = "Test Stub"
+    @priority = "3"
+    test AssetPublisherRejectBlog {
```

Comment:
Hi Kristin! Could you, please, change the test name to "DraftBlogIsNotDisplayed" and change the test description to "Verify that a Draft Blog Entry is not displayed on Asset Publisher"? Thank you! :)

Reply:
Stub is updated!